server/src/main/java/com/linecorp/centraldogma/server/metadata/Token.java

```diff
@@ -122,11 +125,11 @@ public String secret() {
     }
 
     /**
-     * Returns whether this token has administrative privileges.
+     * Returns whether this token has system administrative privileges.
      */
     @JsonProperty
-    public boolean isAdmin() {
-        return isAdmin;
+    public boolean isSystemAdmin() {
```

Comment:
Note: 
1. While releasing, if a user tries to access a token via UI served by old servers to a new server he/she may encounter errors.
2. If a new token is registered while releasing, old servers may not be able to access it
3. Regarding rollback plan: if a new token is registered we need to clean up the token before performing rollback. Having said this, I'm not sure if there is an easy way to detect this at the moment.

Reply:
> While releasing, if a user tries to access a token via UI served by old servers to a new server he/she may encounter errors.

That's correct. 👍 We need to let them refresh the page after finishing the rolling restart.

> If a new token is registered while releasing, old servers may not be able to access it

That's correct. 👍 I thought it should be okay, but it probably shouldn't be. 😓 
Let me create a separate commit to solve this issue. 

> Regarding rollback plan:

I'm implementing a client that migrates `admin` to `systemAdmin`.
Let me also implement the client the vise-versa.

Reply:
@jrhee17 Here it is. 😉 
https://github.com/line/centraldogma/pull/1056